README.md

```diff
@@ -4,7 +4,7 @@
 Creators Update. Anything may happen, be prepared. I'll look into relevant
 changes and recent issues when the update is released / pushed on to everyone.
 
-This project collects Powershell scripts which help to *debloat* Windows 10,
+This project collects PowerShell scripts which help to *debloat* Windows 10,
```

Comment:
Made consistent with all other instances in the file (and correct vs Microsoft promotion)